met-api/src/met_api/services/staff_user_service.py

```diff
@@ -100,29 +100,26 @@ def _render_email_template(user: StaffUserModel):
         )
         return subject, body, args
 
-    # TODO: Replace this method with one that uses composite roles, if necessary
-    # @staticmethod
-    # def attach_roles(user_collection):
-        # """Attach keycloak groups to user object."""
-        # group_user_details = KEYCLOAK_SERVICE.get_users_groups(
-        #     [user.get('external_id') for user in user_collection])
-
-        # for user in user_collection:
-        #     # Transform group name from ADMINISTRATOR to Administrator
-        #     # TODO etc;Arrive at a better implementation than keeping a static list
-        #     # TODO Probably add a custom attribute in the keycloak as title against a group?
-        #     groups = group_user_details.get(user.get('external_id'))
-        #     user['groups'] = ''
-        #     if groups:
-        #         user['groups'] = [GROUP_NAME_MAPPING.get(group, '') for group in groups]
-        #         if Groups.IT_ADMIN.value in user['groups']:
-        #             user['main_group'] = Groups.IT_ADMIN.value
-        #         elif Groups.TEAM_MEMBER.value in user['groups']:
-        #             user['main_group'] = Groups.TEAM_MEMBER.value
-        #         elif Groups.REVIEWER.value in user['groups']:
-        #             user['main_group'] = Groups.REVIEWER.value
-        #         else:
-        #             user['main_group'] = user['groups'][0]
+    @staticmethod
+    def attach_roles(user_collection):
+        """Attach keycloak composite roles to user object."""
+        user_roles = KEYCLOAK_SERVICE.get_users_roles(
+            [user.get('external_id') for user in user_collection])
+        for user in user_collection:
+            # TODO etc;Arrive at a better implementation than keeping a static list
+            # TODO Probably add a custom attribute in the keycloak as title against a group?
```

Comment:
I initially presumed that since we removed the dependency on the tenant attribute, yet opted to maintain the existing comment unchanged. 
Following your suggestion, I have now removed it.